debian/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-slim
+
+ENV PYTHONUNBUFFERED=1
+
+# Work in the /opt directory to persist the pip-compiled requirement list for
+# child images to use.
+WORKDIR /opt
+
+# Update OpenSSL and certificates
+RUN set -eux \
+    && apt-get update \
+    && apt-get install --yes --only-upgrade openssl ca-certificates \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+COPY base-requirements.in ./
+
+RUN set -eux \
```

Comment:
You're not installing the build tools to compile gevent here. Does that even work?

Reply:
Hmm, I have a hard time envisioning a scenario where we need that ability. Have we used it in the past?

Reply:
Yeah, looks like we can use pre-built wheels for this image.